packages/vats/src/proposals/upgrade-vaultFactory-proposal.js

```diff
@@ -0,0 +1,124 @@
+/**
+ * @file this core-eval proposal is a generic and reusable script for executing
+ *   a Vault Factory null upgrade. In contrast, upgrade-vaults.js is a specific
```

Comment:
a "null upgrade" is one where the contract code doesn't change. This one uses the bundleRef from the provided `contractRef`, so it will do an actual upgrade.
```suggestion
 * @file this core-eval proposal is a generic and reusable script for executing
 *   a Vault Factory upgrade. In contrast, upgrade-vaults.js is a specific
```